tcms/testruns/static/testruns/js/get.js

```diff
@@ -282,6 +300,36 @@ function renderTestExecutionRow(testExecution) {
     template.find('.test-execution-status-icon').addClass(testExecutionStatus.icon).css('color', testExecutionStatus.color)
     template.find('.test-execution-status-name').html(testExecutionStatus.name).css('color', testExecutionStatus.color)
 
+    template.find('textarea')[0].id = `comment-for-testexecution-${testExecution.id}`
```

Comment:
All of this needs to be hidden/not executed when the user doesn't have permissions to add a comment, see https://github.com/kiwitcms/Kiwi/pull/2029/files#diff-9443fe8c4b1d877f005483a674b1a59a740c4fd28bede5916efb1ff19108ad8bR353

Reply:
you link is not valid and just shows me the files in this PR. I can't see what you are trying to show me

Reply:
https://github.com/kiwitcms/Kiwi/pull/2029/files#diff-9443fe8c4b1d877f005483a674b1a59a740c4fd28bede5916efb1ff19108ad8bR353

Reply:
that's the same link
![image](https://user-images.githubusercontent.com/18421997/97975518-f0c8e480-1dd1-11eb-9008-229b77c1221f.png)

Reply:
well it doesn't want to expand. See get.js line 353 in that PR

Reply:
yes, I have done something more or less the same. it is already pushed